test/index.js

```diff
@@ -181,7 +184,7 @@ describe('Klaviyo', function () {
         .end(done);
     });
 
-    it('should override confirmOptin setting if provided', function(done){
+    it('should override confirmOptin and listId setting if manually provided', function(done){
```

Comment:
I don' think this test is testing the right thing. How do you know it's picking up from options and not settings? both are `baVTu8`. Should delete `settings.listId;` in this.